### ISSUE TITLE
elevation units

### ISSUE THREAD

**Issue by systemcrash** [claytonjn/hass-circadian_lighting#140]
https://github.com/claytonjn/hass-circadian_lighting/wiki/Advanced-Configuration#component-configuration-variables

It's a float, but what does the float represent? 
metres (like all sane people), or what?

**Comment by claytonjn** [comment]
Yes, meters. I'll update the documentation, thanks for pointing that out.
